### ISSUE TITLE
Automatically generated nginx resolver setting is invalid when /etc/resolv.conf contains IPv6 addresses

### ISSUE THREAD

**Issue by ldruschk** [jitsi/docker-jitsi-meet#1437]
Hi,

the code for generating the nginx resolver setting when the setting is not explicitly specified can be found here: https://github.com/jitsi/docker-jitsi-meet/blob/53b265455661fe0a6fc986ad589d6ecfd911847c/web/rootfs/etc/cont-init.d/10-config#L68

When the `/etc/resolv.conf` contains IPv6 addresses, the resulting config snippet looks as follows:
```
resolver 127.0.0.11 2a03:4000:0:1::e1e6
```

This results in the following error, leading to nginx not being able to start:
```
nginx: [emerg] invalid port in resolver "2a03:4000:0:1::e1e6" in /config/nginx/nginx.conf:30
```

According to the documentation IPv6 resolvers must be specified by surrounding the addresses with brackets: https://nginx.org/en/docs/http/ngx_http_core_module.html#resolver

When manually changing the setting to the following, nginx starts up normally
```
resolver 127.0.0.11 [2a03:4000:0:1::e1e6]
```

**Comment by saghul** [comment]
What environment is that?

**Comment by cdauth** [comment]
Do I see it right that this has not made it into the stable release yet?

**Comment by saghul** [comment]
The commit has been in 9 stable release, since 8044.

**Comment by cdauth** [comment]
Ah, I got confused since I didn't see it in the release notes and the commit in the pull request didn't show any branches/commits (it seems that the pull request doesn't contain the same commit as the master branch).

Anyways, I am running into this error message with the current `stable` image. The `/etc/resolv.conf` file inside the container looks like this:
```
nameserver 127.0.0.11
nameserver fd63:2af7:9f8b:ef5a:1::1
nameserver fe80::216:3eff:fef4:3236%11
options ndots:0
```

Which results in the following nginx config:
```
resolver 127.0.0.11 [fd63:2af7:9f8b:ef5a:1::1] [fe80::216:3eff:fef4:3236%11];
```

This results in the following error message:
```
nginx: [emerg] invalid IPv6 address in resolver "[fe80::216:3eff:fef4:3236%11]" in /config/nginx/nginx.conf:30
```

I have found an nginx issue that might explain this: https://trac.nginx.org/nginx/ticket/2509

It seems like the /etc/resolv.conf file inside the container is the same as on my system, so as a workaround I guess I have to investigate how that address ends up there in the first place and get rid of it.
